lib/driver.js

```diff
@@ -393,8 +393,19 @@ class EspressoDriver extends BaseDriver {
     // start an avd, set the language/locale, pick an emulator, etc...
     // TODO with multiple devices we'll need to parameterize this
     await helpers.initDevice(this.adb, this.opts);
+    this.caps.setAnimation = this.caps.setAnimation || false;
```

Comment:
maybe it makes sence to extract this logic into a separate method as it's getting a but more lengthy and complicated

Reply:
extracted into a function